crypto/ec/ec_curve.c

```diff
@@ -3170,17 +3178,30 @@ static EC_GROUP *ec_group_new_from_data(OSSL_LIB_CTX *libctx,
     params = (const unsigned char *)(data + 1); /* skip header */
     params += seed_len;         /* skip seed */
```

Comment:
shift this line down and then dont do the step backwards part in the group_full_init